### PR TITLE
lookup keys in puppet class form visual changes

### PR DESCRIPTION
**Pull request by abenari**
Make lookup keys list in puppet class form look more like the rest of the product, and improve the form usability.

### CODE REVIEW

**Comment by domcleal** [comment]
Nice improvement, that looks a lot more consistent.  Another issue (since bs3?) is the fields in the form seem quite narrow, particularly for larger matchers & values, inline help text and field labels.  Could you see about making it wider?

**Comment by abenari** [comment]
@domcleal fixed the fields size.

**Comment by domcleal** [comment]
@abenari the matcher value/match fields at the bottom of the form are unchanged, but the others look good

File: app/helpers/layout_helper.rb [hunk 1]
```diff
@@ -187,6 +187,7 @@ def form_to_submit_id f
 
   def submit_or_cancel f, overwrite = false, args = { }
     args[:cancel_path] ||= send("#{controller_name}_path")
+    content_tag(:div, :class => "clearfix") do
     content_tag(:div, :class => "form-actions") do
```
Comment (domcleal) [inline app/helpers/layout_helper.rb]: indentation

File: app/views/puppetclasses/_form.html.erb [hunk 2]
```diff
@@ -70,16 +69,18 @@
       <div class="undo-smart-vars alert alert-info hide">
         <%= _('Undo remove') %>
       </div>
-      <ul class="nav nav-pills nav-stacked smart-var-tabs col-md-3" data-tabs="pills">
+      <div class="smart-var-left col-md-3">
+      <ul class="nav nav-pills nav-stacked smart-var-tabs" data-tabs="pills">
```
Comment (domcleal) [inline app/views/puppetclasses/_form.html.erb]: indentation

**Comment by abenari** [comment]
@isratrade mind reviewing?

**Comment by domcleal** [comment]
:+1: from me.  Note, we need to file a redmine for this before merging + squash commits.

**Comment by isratrade** [comment]
@abenari, :+1: from me. What happened to the blue arrow when a lookup value is selected and the the active rectangle table is blue?

**Comment by abenari** [comment]
[test]

**Comment by domcleal** [comment]
Merged as 8527c9f for Foreman 1.5.0, thanks @abenari!

**Pull request closed without merging**